### PR TITLE
fix: enriched Request object is not being passed to windows fetch

### PR DESCRIPTION
**Pull request by davewoodward**
- Added test to verify Request object is passed.
- Added assertion to verify that the Request object contains the Authorization header.
- Updated eslint-config-google dependency and added indent rule configuration to eslint configuration..
- Updated webpack-dev-server dependency.

### CODE REVIEW

File: src/salte-auth.utilities.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ class SalteAuthUtilities {
             promises.push(interceptor(request));
           }
           return Promise.all(promises).then(() => {
-            return fetch.call(this, input, options);
+            return fetch.call(this, request, options);
```
Comment (cecilia-sanare) [inline src/salte-auth.utilities.js]: Can you remove options?

File: .eslintrc.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ plugins:
 parserOptions:
   sourceType: module
 rules:
+  indent: [2, 2]
```
Comment (cecilia-sanare) [inline .eslintrc.yml]: Can you make this like the rule below it using the yaml array syntax with error and 2.

**Review by cecilia-sanare — CHANGES_REQUESTED** [review]

File: src/salte-auth.utilities.js (hunk 1 above)
Comment (davewoodward) [reply src/salte-auth.utilities.js]: Good catch!

File: .eslintrc.yml (hunk 2 above)
Comment (davewoodward) [reply .eslintrc.yml]: You got it!

**Comment by davewoodward** [comment]
Requested changes have been made.  I also fixed the SalteAuth instantiation example on the readme by prefixing it with "salte".

File: README.md [hunk 3]
```diff
@@ -76,7 +76,7 @@ We also support HTML Imports:
 import { SalteAuth } from '@salte-io/salte-auth';
 
 // Configure SalteAuth with Auth0's url and client id.
-const auth = new SalteAuth({
+const auth = new salte.SalteAuth({
```
Comment (cecilia-sanare) [inline README.md]: Why’d you change this? It was correct before.

**Review by cecilia-sanare — CHANGES_REQUESTED** [review]

**Review by cecilia-sanare — APPROVED** [review]

**Pull request merged**